v2.0/partitioning.md

```diff
@@ -0,0 +1,371 @@
+    PARTITION current VALUES FROM ('2018-08-15') TO (MAXVALUE));
+~~~
+
+### Unpartition a Table
+
+You can remove the partitions on a table by using the `PARTITION BY NOTHING` syntax with the `ALTER TABLE` command:
+ 
+~~~ sql
+> ALTER TABLE students PARTITION BY NOTHING;
+~~~
+
+## Licensing Requirements
+
+You require an enterprise license to partition or repartition a table, and to add or edit the zone configurations of a partition or an index. To enable partitioning, [set the Trial or Enterprise License Key](enterprise-licensing.html#set-the-trial-or-enterprise-license-key).
+
+### Expired License
+
+The following features don't work with an expired license:
+
+- Creating new table partitions or adding new zone configurations for partitions
+- Changing the partitioning scheme on any table or index
+- Changing the zone config for a partition
+
+However, the following features continue to work even with an expired enterprise license:
+
+- Querying a partitioned table (for example, `SELECT foo PARTITION`)
+- Inserting or updating data in a partitioned table
+- Dropping a partitioned table
+- Unpartitioning a partitioned table
+- Making non-partitioning changes to a partitioned table (for example, adding a column/index/foreign key/check constraint) 
+
+## Locality–Resilience Tradeoff
+
+There is a tradeoff between making reads/writes fast and surviving failures. Consider a partition with three replicas of `roachlearn.students` for Australian students. If only one replica is pinned to an Australian datacenter, then reads may be fast (via leases follow the workload) but writes will be slow. If two replicas are pinned to an Australian datacenter, then reads and writes will be fast (as long as the cross-ocean link has enough bandwidth that the third replica doesn’t fall behind). If those two replicas are in the same datacenter, then the loss of one datacenter can lead to data unavailability, so some deployments may want two separate Australian datacenters. If all three replicas are in Australian datacenters, then three Australian datacenters are needed to be resilient to a datacenter loss.
+
+## How CockroachDB's Partitioning Differs from Other Databases
+
+Other databases use partitioning for three additional use cases: secondary indexes, sharding, and bulk loading/deleting. CockroachDB addresses these use-cases not by using partitioning, but in the following ways:
+
+- **Changes to secondary indexes:** CockroachDB solves these changes through online schema changes. Online schema changes are a superior feature to partitioning because they require zero-downtime and eliminate the potential for consistency problems. 
+- **Sharding:** CockroachDB automatically shards data as a part of its distributed database architecture. 
+- **Bulk Loading & Deleting:** CockroachDB does not have a feature that supports this use case as of now. 
+
+## See Also
+
+[Other Cockroach Commands](cockroach-commands.html)
```

Comment:
This feels strange, given that this page is not focused only on a cockroach command. Can you think of other relevant docs tied to this page?